lib/foreman_maintain/runner.rb

```diff
@@ -38,5 +43,23 @@ def ask_about_offered_steps(steps)
         @reporter.on_next_steps(self, steps)
       end
     end
+
+    def mock_skip_class(step)
+      Class.new(step.class.superclass) do
```

Comment:
Okay, so you are suggesting we do not show `[SKIPPED]` status for the single(binary) procedure?

In case of multi-options, we are not showing `[SKIPPED]` status. So to keep both of them consistent, we can just skip the check without showing anything on the next line and moving on to the next check. Makes sense?